src/device/pf_cmina.c

```diff
@@ -929,6 +938,15 @@ int pf_cmina_dcp_get_req(
    return ret;
 }
 
+int pf_cmina_get_file_directory(
```

Comment:
Is it a fatal error to not supply a file_directory? And what happens in a release build with assertions disabled? It seems that the return value of this function is not checked.

Reply:
Moved check to configuration validation.